### ISSUE TITLE
This addon will not work in Firefox 57, scheduled to be released on 2017-11-28

### ISSUE THREAD

**Issue by JBPhoenix** [JBPhoenix/ff-hide-plugin-bar#4]
Mozilla have recently announced a timeline for the phasing out of addons using the existing extension frameworks. Unfortunately and unexpectedly for me, this includes the addon sdk/jetpack used by this extension:
https://blog.mozilla.org/addons/2016/11/23/add-ons-in-2017/

As I understand it, there is no way to replicate the functionality of this addon using "WebExtensions" right now - and I'm not sure there will be (would love to be wrong!).

tl;dr - This addon is going to break in a year. I'll do what I can, but please shout if you see this and have any bright ideas.
